src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -302,7 +303,8 @@ public String getDefaultBranch() throws IOException, InterruptedException {
         try {
             response = getRequest(url);
         } catch (FileNotFoundException e) {
-            LOGGER.fine(String.format("Could not find default branch for %s/%s", this.owner, this.repositoryName));
+            LOGGER.log(Level.FINE, "Could not find default branch for {0}/{1}",
+                    new Object[]{this.owner, this.repositoryName});
```

Comment:
or more easily, in Java 8 just prepend `() -> ` to the message

Reply:
Not on Java 8 baseline yet, and iirc JMH shows the array as faster than INDY